rts/Game/Camera.cpp

```diff
@@ -22,6 +22,21 @@ CONFIG(float, EdgeMoveWidth)
 CONFIG(bool, EdgeMoveDynamic)
 	.defaultValue(true)
 	.description("If EdgeMove scrolling speed should fade with edge distance.");
+CONFIG(float, CameraMoveFastMult)
+	.defaultValue(3.0f)
+	.minimumValue(1.0f)
+	.description("The multiplier applied to speed when camera is in movefast state.");
+CONFIG(float, CameraMoveSlowMult)
+	.defaultValue(0.1f)
+	.maximumValue(1.0f)
+	.description("The multiplier applied to speed when camera is in moveslow state.");
+CONFIG(float, CameraFastScale)
+	.defaultValue(10.0f / 3.0f)
+	.description("Scaling for CameraMoveFastMult.");
```

Comment:
What sort of scaling? Why not just make Mult 10 and have no Scale?

Reply:
The thing is that different places (Camera, OverheadCamera, SpringCamera) used different scales for movefast state: 10, 3, and 2. CameraMoveFastMult changes the multiplier which increases the speed of camera movement in all situations. CameraFastScale changes the scaling factor 10, which is used here, springFastScale changes the scaling factor for spring camera only (2).
Does this make sense to you?

Reply:
Cameras having their own mult makes sense. But if the 3 is only used for Overhead, and 10 for a "generic" camera, then how about:
```
CameraMoveFastMult -> default 3
CameraFastScale -> default 10/3
CamSpringFastScale -> default 2/3
```
to
```
CameraMoveFastMult -> default 10
CamOverheadFastScale -> default 3/10
CamSpringFastScale -> default 2/10
```
?

Reply:
makes sense, will do

Reply:
It seems that spring uses 2 different scales (3 and 2)
https://github.com/beyond-all-reason/spring/pull/475/files#diff-929336c760fe2d068ad36f4cd3ace761039308f44199acf5ab5ee2ee420c1284L69
and
https://github.com/beyond-all-reason/spring/pull/475/files#diff-929336c760fe2d068ad36f4cd3ace761039308f44199acf5ab5ee2ee420c1284L99
rescaling would result in either another extra parameter or a magic constant 3/2. I think I will introduce a const for that.

Reply:
ok, I will make 2 settings

Reply:
This works, but if someone starts chaning CameraMoveFastMult, then for example in overhead mode setting the multiplier less than 3 actually slows down while holding shift. Dunno, if this is desired behaviour.

Reply:
This is all messed up, not a fault of your code specifically, but the decoupling of behavior among cameras and the zillion ways they differ in it.

Reply:
Personally I'd use a single multiplier for everything, but the current approach is the best if we want to keep maintainability.